templates/usr/local/bin/occ

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# {{ ansible_managed }}
+
+sudo --user {{ owncloud_user }} php --file /var/www/owncloud/occ "$@"
```

Comment:
This should be templated to point to correct directory where ownCloud is installed.